shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -45,6 +45,7 @@
 static NSString* const kSetSelectionRectsMethod = @"Scribble.setSelectionRects";
 static NSString* const kStartLiveTextInputMethod = @"TextInput.startLiveTextInput";
 static NSString* const kUpdateConfigMethod = @"TextInput.updateConfig";
+static NSString* const kOnPointerMoveMethod = @"TextInput.onPointerMove";
```

Comment:
Nit maybe `kOnInteractiveKeyboardPointerMoveMethod = @"TextInput.onPointerMoveForInteractiveKeyboard"`, to be more specific.

Reply:
Just updated, I'll make sure to try and add Interactive Keyboard more in the future, definitely would help with clarity!